src/components/nav-bar/nav-bar.tsx

```diff
@@ -7,7 +7,7 @@ import { mergeProps } from '../../utils/with-default-props'
 const classPrefix = `adm-nav-bar`
 
 export type NavBarProps = {
-  back?: string | null
+  back?: ReactNode | null
```

Comment:
```suggestion
  back?: ReactNode
```

null 也在 ReactNode 里的